src/test/rustdoc-ui/display-output.rs

```diff
@@ -1,9 +1,15 @@
+// Test that `--show-output` has an effect and `allow(unused)` can be overriden.
+
 // check-pass
-// compile-flags:-Zunstable-options --display-doctest-warnings --test
+// edition:2018
+// compile-flags:-Zunstable-options --test --test-args=--show-output
```

Comment:
An example of why `-Z my-option` is better than `-Z unstable-options` ;)

```suggestion
// compile-flags:--test --test-args=--show-output
```